CHANGELOG.md

```diff
@@ -3,6 +3,17 @@ Change Log
 
 This file documents all notable changes to Peggy.
 
+NextVersion
+-----------
```

Comment:
```suggestion
Unreleased
----------
```

Reply:
I like that.  Should I change the branch name also?

Reply:
Yeah, I decided to change the branch name to `unreleased`.  Editing above.

Reply:
It seems that we have already discussed this, but still, why a separate branch? Now you have removed `1.3` and all associated PR automatically closed, which is not very good. Releasing of version does not mean that unmerged PRs not actual anymore. It will be better to make PRs against `main` branch, which is always exist

Reply:
In particular, I want the docs that people see on https://peggyjs.org/ to be for the currently-released version - that's currently served from main, but we could make a separate docs branch if we're careful about how it gets updated.  I have a slight preference that the code that people see by default is the currently-releaased version, but I don't care about that as much.

I agree with you that all of those issues closing when I merged the 1.3 branch was not good.

Note that I'm completely open to changing how we move forward here, and didn't realize we weren't on the same page yet.  Sorry about that.

Reply:
A separate branch for docs definitely a better solution. Maybe even just have a special tag for the current docs and a CI task that will rebuild doc when that tag moved to a new commit.

> Note that I'm completely open to changing how we move forward here, and didn't realize we weren't on the same page yet. Sorry about that.

Well, it wasn't that bad (only very unusual), I realized flaws of this approach only when my PRs was accidently closed

Reply:
If we've got a `docs` branch, I think it would look like:

- If you want to make a change to peggyjs.org now (e.g. better examples that still work on the current version), you make the PR to `docs`, then cherry-pick the commit to `main`
- If you want to make a change to the docs for the unreleased version (e.g. new feature that doesn't work on the current version), you make the PR to `main`
- When we do a release, we merge `main` onto `docs` and deal with whatever conflicts come up.  If we're careful about cherry-picking from `docs` to `main`, hopefully that won't be too bad

Then, for new work:
- PR goes to `main`
- No extra merge work at release time

That leaves the `README.md` looking like the future version, which I don't like.  We could solve that by deleting most of that doc, leaving a pointer to `https://peggyjs.org/documentation.html`, leaving something that looks more like `https://peggyjs.org/index.html`.  This would have the benefit of removing our need to keep things synchronized.

Reply:
I could prototype that if it would help?

Reply:
Yes, I think we should go in that direction